<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkImageNavigatorViewControls</class>
 <widget class="QWidget" name="QmitkImageNavigatorViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>514</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QmitkSliderNavigatorWidget" name="m_SliceNavigatorTransversal" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QmitkSliderNavigatorWidget" name="m_SliceNavigatorSagittal" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QmitkSliderNavigatorWidget" name="m_SliceNavigatorFrontal" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="QmitkSliderNavigatorWidget" name="m_SliceNavigatorTime" native="true"/>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>429</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkSliderNavigatorWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSliderNavigatorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">QmitkDataStorageComboBox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
